frontend/src/forms/ParcelDetailForm.test.tsx

```diff
@@ -97,7 +108,7 @@ describe('ParcelDetailForm', () => {
       agencyId: 1,
       address: {
         line1: 'addressval',
-        cityId: 1,
+        cityId: '1',
```

Comment:
I'm curious why we use strings here?

Reply:
Most likely this comes from a `<select>`. Selects always return their value in string form, or so I've seen.

https://www.w3schools.com/tags/tag_option.asp